suite-common/bluetooth/src/bluetoothReducerCreator.ts

```diff
@@ -0,0 +1,133 @@
+import { AnyAction, Draft } from '@reduxjs/toolkit';
+
+import { createReducerWithExtraDeps } from '@suite-common/redux-utils';
+import { deviceActions } from '@suite-common/wallet-core';
+
+import {
+    bluetoothAdapterEventAction,
+    bluetoothConnectDeviceEventAction,
+    bluetoothDeviceListUpdate,
+    bluetoothScanStatusAction,
+} from './bluetoothActions';
+
+export type BluetoothScanStatus = 'running' | 'done' | 'error';
```

Comment:
I find `done` to confusing. It seems as a final status from which there's no way back. What about `idle` instead?
```suggestion
export type BluetoothScanStatus = 'idle' | 'running' | 'error';
```

Reply:
fixed